fe1-web/src/features/rollCall/components/RollCallOpen.tsx

```diff
@@ -48,10 +50,35 @@ const RollCallOpen = ({
   const hasSeed = RollCallHooks.useHasSeed();
   const toast = useToast();
   const navigation = useNavigation<NavigationProps['navigation']>();
+  const [popTokens, setPopTokens] = useState<PublicKey[]>(scannedPopTokens || []);
 
   const [popToken, setPopToken] = useState('');
   const [hasWalletBeenInitialized, setHasWalletBeenInitialized] = useState(hasSeed());
 
+  const updateAttendeeList = useCallback(() => {
+    // If it is the organizer, we add his token to the list (he automatically attends the roll call)
+    const personalPopToken = isOrganizer && popToken !== '' ? [new PublicKey(popToken)] : [];
+    // If it comes from the scanner then we take this list, otherwise we try if it has already been set
+    const defaultAttendeesList = scannedPopTokens || rollCall.attendees || [];
+    // if the list is empty, we take the personalPopToken (which is non-empty if the user is the organizer)
+    const attendeesList = defaultAttendeesList.length > 0 ? defaultAttendeesList : personalPopToken;
+
+    setPopTokens(attendeesList);
+
+    // Stores the already scanned tokens in case we go to a new screen (different from the scanner)
+    const updatedRollCall = {
+      ...rollCall.toState(),
+      attendees: attendeesList.map((key) => key.toState()),
+    };
+
+    dispatch(updateRollCall(updatedRollCall));
```

Comment:
This is problematic for the same reason described here: https://github.com/dedis/popstellar/pull/1643#issuecomment-1579422050

In fe-1, the redux store should only ever be updated because of network messages..